### DISCUSSION TITLE
Shiny.NFC support to version 3?

### DISCUSSION THREAD

**Discussion by gheorghedarle** [shinyorg/shiny#1422] [Q&A] [answered]
First of all thanks for the great library, is there any plans to support Shiny.NFC in version 3? 
I've tried on Xamarin.Forms with the prerelease version of Shiny.NFC and Shiny.Core 3 and I received this exception on this line **builder.Services.AddNfc()** in the Android project:
**System.TypeLoadException: 'Could not resolve type with token 01000042 from typeref (expected class 'Shiny.ServiceExtensions' in assembly 'Shiny.Core, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null')'**

Shiny 2.7.3
Shiny.Core 3.2.2
Shiny.NFC 3.0.0-alpha-0402

**Comment by aritchie** [comment, marked as answer]
NFC is gone and was not brought in to v3
